core/icmp-jna/c-ping/.gitignore

```diff
@@ -1,21 +0,0 @@
-*.cache
```

Comment:
Don't these exclusions need to stay in place to exclude the output files of the C build from getting checked in?